tests/yaml/multipass-vs-match.yaml

```diff
@@ -0,0 +1,386 @@
+#!tools/lou_checkyaml
+#
+# Comparison of match and multi-pass opcodes
+# ::::::::::::::::::::::::::::::::::::::::::
+#
+# Copyright © 2019 by Bert Frees
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+# ..............................................................................................
+
+# use "display" also to define common part for all tests
+display: |
```

Comment:
This feels really weird. I know you implemented this in an other PR, but now that I see ti in use it feels kinda wrong.

Could you not instead just define a common table that has all the stuff that you now have in the `display`? I guess the downside of that is that you cannot define it all in the YAML test. With your enhancement you can. Well, OK, I guess you can't have it all. So, let's leave it then. Just make sure to leave the comment above :-)

Reply:
Yes but then it's less visible, and that's exactly the purpose of this test, it serves as documentation. I know it's wrong, and when the translation and display phases have been separated this won't even be possible anymore. But still, for now...

I'll put an extra comment to make clear it is a hack.